library/core/resource_loader/src/main/java/org/quiltmc/qsl/resource/loader/impl/ResourceLoaderImpl.java

```diff
@@ -549,8 +549,20 @@ private static List<ResourcePack> findUserStaticPacks() {
 					if (pathAsFile.toPath().toString().endsWith(".zip")) {
 						returnList.add(new ZipResourcePack(n, pathAsFile, false));
 					} else {
-						if (!STATIC_USERSPACE_IGNORED_FILES.contains(pathAsFile.getName())) {
-							//Ignores filesystem helper files, currently ignoring .DS_Store and thumbs.db
+						boolean fsHelper = false;
+						for (String fs : STATIC_USERSPACE_IGNORED_FILES) {
+							// Implementation detail: ._* and (some) *nix-derivative OSes (macOS, Linux distros, etc)
+							// ._* is a rare *nix filesystem helper file created to store file information normally placed in an extended attribute
+							// on HFS+ (Apple Native, typically found under macOS) or UFS (other *nix OSes) filesystems when interfacing with a filesystem
+							// that does NOT support such extended attributes (for instance, a FAT32 drive). Since these can have names of arbitrary lengths,
+							// the only way to detect them is via the shared prefix.
+							// Source: https://apple.stackexchange.com/a/14981
+							if (fs.equalsIgnoreCase(pathAsFile.getName()) || pathAsFile.getName().startsWith("._")) {
```

Comment:
There is no benefit I see to repeating the `startsWith` check multiple times within the loop; guessing this is an oversight?

Reply:
Yeah, it is—I'll move it outside of the loop in a commit in a moment

Reply:
Fixed as of latest commit.